### ISSUE TITLE
Feature Request - Clone All Org Repos

### ISSUE THREAD

**Issue by eliyastein** [mschwager/gitem#3]
Per the reddit thread: Would love to have the ability to clone every repo from an organization and also every repo belonging to every user of that organization.

**Comment by mschwager** [comment]
Hey! Thanks for the suggestion! I could add some functionality around just printing the user's/organization's repository names. That could then be coordinated with `git clone` to get this functionality.

Another option is to work some bash magic, something like:

```bash
$ for repository in $(gitem --verbose organization <organization> | grep <organization> | grep "Github URL" | cut -d " " -f 5); do git clone $repository; done
```

Which would work similarly for user's as well.

**Comment by eliyastein** [comment]
True... if gitem output was json, then it would be even easier to pipe through jq as well.

**Comment by mschwager** [comment]
I will investigate different output formats!

**Comment by mschwager** [comment]
This feature's available in `0.8.0`: https://pypi.org/project/gitem/0.8.0/
